tests/validation/test_validate.py

```diff
@@ -80,14 +79,46 @@ def test_validate_error_contains_href(self) -> None:
                 assert get_opt(item.get_self_href()) in str(e)
                 raise e
 
+    @pytest.mark.vcr()
+    def test_validate_all_deprecated_dict_arg(self) -> None:
+        catalog = TestCases.case_1()
+        catalog.fully_resolve()
```

Comment:
Did you check to see if these are still required?

Reply:
Ah, yes, I was able to remove the call to `fully_resolve`.